.storybook/blocks/PropertiesTable.jsx

```diff
@@ -0,0 +1,205 @@
+                            className="spectrum-Icon spectrum-UIIcon-Cross100 spectrum-Icon--medium spectrum-CloseButton-UIIcon"
+                            focusable="false"
+                            aria-hidden="true"
+                            role="img"
+                            xmlns="http://www.w3.org/2000/svg"
+                            viewBox="0 0 8 8"
+                        >
+                            <path d="m5.238 4 2.456-2.457A.875.875 0 1 0 6.456.306L4 2.763 1.543.306A.875.875 0 0 0 .306 1.544L2.763 4 .306 6.457a.875.875 0 1 0 1.238 1.237L4 5.237l2.456 2.457a.875.875 0 1 0 1.238-1.237z"></path>
+                        </svg>
+                    </button>
+                </div>
+            </div>
+		</Container>
+	);
+};
+
+/**
+ * Write the provided text to the clipboard using the Clipboard API and
+ * show a toast message to indicate the operation was successful or not.
+ *
+ * @param {string} text
+ */
+export function writeTextToClipboard(text) {
+	function updateMessage(toast, content) {
+		const container = toast.querySelector(".spectrum-Toast-content");
+		if (container) container.innerHTML = content;
+		else return;
+	}
+
+	function toggleToast(toast, open = true) {
+		toast.classList.toggle("is-open", open);
+	}
+
+	navigator.clipboard.writeText(text).then(
+		() => {
+			const toast = document.getElementById("clipboard-success");
+			const content = `Copied <em>${text}</em> to clipboard`;
+
+			updateMessage(toast, content);
+			toggleToast(toast);
+			setTimeout(() => {
+				toggleToast(toast, false);
+			}, 3000);
+
+			console.log(toast, content);
+		},
+		() => {
+			const toast = document.getElementById("clipboard-failure");
+			const content = `Failed to copy <em>${text}</em> to clipboard`;
+
+			updateMessage(toast, content);
+			toggleToast(toast);
+			setTimeout(() => {
+				toggleToast(toast, false);
+			}, 3000);
+		},
+	);
+}
+
+/**
+ * Displays the current version number of the component. The version is read from
+ * the component's parameters, where it was sourced from the package.json file.
+ *
+ * Also displays a component status of "deprecated" if it is set in the story's
+ * parameters.
+ *
+ * Usage of this doc block within MDX template(s):
+ *  <PropertiesTable />
+ */
+export const PropertiesTable = () => {
+	const storyMeta = useOf("meta");
+
+	const packageJson = storyMeta?.csfFile?.meta?.parameters?.packageJson ?? {};
+	const metadata = storyMeta?.csfFile?.meta?.parameters?.metadata ?? {};
+
+	if (!packageJson?.name) return;
+	if (!metadata?.modifiers) return;
+
+	document.addEventListener("DOMContentLoaded", () => {
+		// Capture all the table body cells and add a click event to copy the text to the clipboard.
+		const tableBodyCells = document.querySelectorAll(
+			".docblock-properties-table tbody td",
+		);
+		tableBodyCells.forEach((cell) => {
+			cell.addEventListener("click", () => {
+				if (cell.textContent) writeTextToClipboard(cell.textContent);
+			});
+		});
+	});
```

Comment:
Instead of an event listener on each cell, this could be a single event listener on the parent, that bubbles up from the click event (then also check `event.target.tagName === "TD"`).